addons/binding/org.openhab.binding.gpio/OSGI-INF/GPIOHandlerFactory.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2014-2016 by the respective copyright holders.
```

Comment:
It seems that this file has a missing, outdated or otherwise non standard copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.

Reply:
When I run `mvn license:format` the following error occurs:
`[ERROR] Failed to execute goal com.mycila:license-maven-plugin:3.0:format (default-cli) on project org.openhab.binding.gpio: Execution default-cli of goal com.mycila:license-maven-plugin:3.0:format failed: Cannot read header document licenses/epl-2.0/xml-header-style.xml. Cause: Resource licenses/epl-2.0/xml-header-style.xml not found in file system, classpath or URL: no protocol: licenses/epl-2.0/xml-header-style.xml`

How can I fix this?

Reply:
I think [https://github.com/openhab/openhab2-addons/pull/5060](https://github.com/openhab/openhab2-addons/pull/5060) will fix this